resources/views/site/teamspeak.blade.php

```diff
@@ -36,6 +36,25 @@
             </ul>
         </div>
         <div class="col-sm-2">
+        </div>
+
+            </div> 
+<hr>
+    <center><h2>Link your TeamSpeak3 PC Client UUID to your ZTL Controller Profile to assign your roles:</h2></center>
```

Comment:
Recommend you shorten this heading... maybe "Howto Link TS3 Client/Auto-Assign Roles:"

It's much longer than the previous heading - this is just a suggestion for look/feel.

Reply:
I feel like this title best explains what's going on for a user that may not understand what exactly it happening/they are doing. They may not be familiar with TS3, or why they need to link their account.